frontend/src/ui-components/AcmTable/AcmTable.tsx

```diff
@@ -417,6 +417,9 @@ const outerDivClass = css({
   display: 'block',
 })
 const tableClass = css({
+  '& tbody > tr > [data-label="Engine"]': {
```

Comment:
This seems specific to our table rather than generic for `AcmTable`. Could you specify this only in the policy tables instead?

Reply:
Let me try